container_registry/container_analysis/src/sample/create_note.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Samples for the Container Analysis golang libraries: https://cloud.google.com/container-registry/docs/container-analysis
```

Comment:
No need for this comment on every single package declaration in this file - one file is enough. Might want to create a `doc.go` which just has the package comment + declaration.

Also, nit, should start with `Package sample`. And no need to say `golang`. :) https://github.com/golang/go/wiki/CodeReviewComments#package-comments

Reply:
Ok, I removed the comment from all but the samples_test file. How does that look?